controllers/operandrequest/reconcile_operator.go

```diff
@@ -54,18 +55,22 @@ func (r *Reconciler) reconcileOperator(ctx context.Context, requestInstance *ope
 		registryKey := requestInstance.GetRegistryKey(req)
 		registryInstance, err := r.GetOperandRegistry(ctx, registryKey)
 		if err != nil {
-			if apierrors.IsNotFound(err) {
-				r.Recorder.Eventf(requestInstance, corev1.EventTypeWarning, "NotFound", "NotFound OperandRegistry NamespacedName %s", registryKey.String())
-				klog.Errorf("failed to find OperandRegistry %s : %v", registryKey.String(), err)
-				requestInstance.SetNotFoundOperatorFromRegistryCondition(registryKey.String(), operatorv1alpha1.ResourceTypeOperandRegistry, corev1.ConditionTrue)
-				mergePatch, _ := json.Marshal(map[string]interface{}{
-					"metadata": map[string]interface{}{
-						"annotations": map[string]interface{}{
-							constant.FindOperandRegistry: "true",
-						},
+			r.Recorder.Eventf(requestInstance, corev1.EventTypeWarning, "NotFound", "NotFound OperandRegistry NamespacedName %s", registryKey.String())
+			klog.Errorf("failed to find OperandRegistry %s : %v", registryKey.String(), err)
+			requestInstance.SetNotFoundOperatorFromRegistryCondition(registryKey.String(), operatorv1alpha1.ResourceTypeOperandRegistry, corev1.ConditionTrue)
+			t := time.Now()
+			formatted := fmt.Sprintf("%d-%02d-%02dT%02d:%02d:%02d",
+					t.Year(), t.Month(), t.Day(),
```

Comment:
lint check failed, I guess it may be caused by using spaces for indentation.
```
controllers/operandrequest/reconcile_operator.go:63: File is not `goimports`-ed with -local github.com/IBM/ (goimports)
					t.Year(), t.Month(), t.Day(),
					t.Hour(), t.Minute(), t.Second())
```
http://prow.purple-chesterfield.com/log?job=check-operand-deployment-lifecycle-manager&id=1390860708563914753

Reply:
Thanks, updated